servicex/values.yaml

```diff
@@ -78,6 +78,8 @@ codeGen:
 transformer:
   pullPolicy: IfNotPresent
   autoscalerEnabled: false
+  cpuLimit: 1
+  cpuScaleThreshold: 1
```

Comment:
Is this a good default value for people to use?

Reply:
Maybe not for the cpuScaleThreshold. After some testing on high-efficiency transformers, I would suggest that 70% provides a reasonable balance between speed of spinning up a new pod and the issue of making too many unused pods.